blocks-common/i-bem/__html/i-bem__html.bemhtml

```diff
@@ -472,6 +472,7 @@ default: {
         var content = apply(this._mode = 'content');
         if(content || content === 0) {
             var isBEM = this.block || this.elem;
+            resetApplyNext();
```

Comment:
Hm... after some consideration, I'm not sure how this could be done. This thing is declared in completely different way in bem-core. It is a keyword in old xjst.